proto/parca/debuginfo/v1alpha1/debuginfo.proto

```diff
@@ -43,7 +43,7 @@ message UploadInfo {
   // build_id is a unique identifier for the debug data
   string build_id = 1;
 
-  // hash is the hash of the debug information file
+  // hash is the hash of the source file that debug information extracted from
```

Comment:
But it's not the hash of the debug information. I can work on the wording. However, it was the mistake previously we made we should store the hash of the source we extracted debug information from. I'm open to suggestions for the wording.

Reply:
Ohhh! I see, I misunderstood what it was, I thought it was `hash(extracted_debug_file)` instead of `hash(original_executable)`. Everything makes sense now 😄